src/api/selection.js

```diff
@@ -35,6 +35,13 @@ function (elementHelper) {
       var range = this.range;
       if(!range) { return; }
 
+      var scribeNodeRange = document.createRange();
+      scribeNodeRange.selectNodeContents(scribe.el);
+      //check if the END of the current selection is within the scribe node
+      if (scribeNodeRange.compareBoundaryPoints(Range.START_TO_END, this.range) < 0){
```

Comment:
AFAICS from [the docs](https://developer.mozilla.org/en-US/docs/Web/API/range.compareBoundaryPoints), this condition is true if the *start* of `this.range` (the current selection) is *after* the *end* of `scribeNodeRange` (i.e. the Scribe container element). That means the condition is `false` if the current selection starts and ends before the present Scribe instance. It also doesn't account for a selection that would overlap with the boundaries of the Scribe container element.

You may want to instead compare the start of the current selection with the start of the Scribe element, and the end of the current selection with the end of the Scribe element?

Unit tests tend to help with this kind of fiddly logic, though you may need to write the code in such a way to make it easier to test said logic.